odoo/addons/base/models/res_users.py

```diff
@@ -1641,7 +1641,7 @@ def _prepare_warning_for_group_inheritance(self, user):
             missing_implied_groups = missing_implied_groups.filtered(
                 lambda g:
                 g.category_id not in (group.category_id | categories_to_ignore) and
-                g not in current_groups_by_category[g.category_id]
+                g not in current_groups_by_category[g.category_id] and (g.category_id if not self.user_has_groups('base.group_no_one') else True)
```

Comment:
Hello @odoo/rd-framework-py I don't know if you want to review this commit. 🙂

Reply:
Done Thanks :+1: